serverlessworkflow/modules/ROOT/pages/cloud/quarkus/build-workflow-images-with-tekton.adoc

```diff
@@ -1 +1,156 @@
+      name: deployment-name
+      type: string
+    - description: url of the git repo for the code of deployment
+      name: git-url
+      type: string
+    - default: pipelines-1.9
+      description: revision to be used from repo of the code for deployment
+      name: git-revision
+      type: string
+    - description: image to be built from the code
+      name: IMAGE
+      type: string
+  tasks:
+    - name: fetch-repository
+      params:
+        - name: url
+          value: $(params.git-url)
+        - name: subdirectory
+          value: ''
+        - name: deleteExisting
+          value: 'true'
+        - name: revision
+          value: $(params.git-revision)
+      taskRef:
+        kind: ClusterTask
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+    - name: build-image
+      params:
+        - name: IMAGE
+          value: $(params.IMAGE)
+        - name: TLSVERIFY
+          value: 'false'
+      runAfter:
+        - fetch-repository
+      taskRef:
+        kind: ClusterTask
+        name: buildah
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+  workspaces:
+    - name: shared-workspace
+----
+
+If you use the Tekton dashboard on Minikube, Kind or on a Kubernets cluster, copy the content of the yam in the pipeline section.
+I you want to skip the UI, save the content of the yaml, with placeholder replaced, in a yaml file called pipeline.yaml and execute
+----
+kubectl apply -f pipeline.yaml
+----
+
+== Create The PipelineRun to execute the pipeline
+
+The next step is to create a pipeline run with the details of your git repository and the desired name of the image.
+
+On OCP in the Pipeline menu, in the item, PipelineRun copy the following yaml file replacing the placeholder with your values
+i.e. `image registry available at image-registry.openshift-image-registry.svc:5000/`
+
+----
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: custom-workflow-image-pipeline-run
+  namespace: custom-namespace
+spec:
+  workspaces:
+    - name: shared-workspace
+      volumeClaimTemplate:
+        spec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 500Mi
+  params:
+    - name: deployment-name
+      value: <your-image-name>
+    - name: git-url
+      value: https://<your repo>.git
+    - name: git-revision
+      value: "main"
+    - name: IMAGE
+      value: "image-registry.openshift-image-registry.svc:5000/custom-namespace/<your-image-name>:latest"
+  pipelineRef:
+    name: custom-workflow-image-pipeline
+----
+You can check the execution of the pipeline in the UI,
+at the end your image will be available at:
+image-registry.openshift-image-registry.svc:5000/custom-namespace/<your-image-name>:latest
+
+On Minikube, Kind or plain Kubernetes cluster you can install the Tekton Dashboard https://tekton.dev/docs/dashboard/
```

Comment:
move it to link

Reply:
e.g. link:{tekton_dashboard_url}[Tekton Dashboard]

Reply:
ok